README.md

```diff
@@ -9,14 +9,125 @@
+    ```dockerfile
+    FROM fairembodied/habitat-challenge:homerobot-ovmm-challenge-2023-dev2
+
+    # install dependencies in the home-robot conda environment
+    RUN /bin/bash -c ". activate home-robot; pip install <some extra package>"
+
+    ADD eval_baselines_agent.py agent.py
+    ADD submission.sh submission.sh
+
+    CMD ["/bin/bash", "-c", ". activate home-robot; export PYTHONPATH=/home-robot/projects/habitat_ovmm:$PYTHONPATH; bash submission.sh"]
+    ```
+1. Build your Docker image using:
+
+    ```
+    docker build . -f docker/ovmm_baseline.Dockerfile -t ovmm_baseline_submission
+    ```
+     
+    *Note:* Please, make sure that you keep your local version of `fairembodied/habitat-challenge:homerobot-ovmm-challenge-2023-dev2` image up to date with the image we have hosted on [dockerhub](https://hub.docker.com/r/fairembodied/habitat-challenge/tags). This can be done by pruning all cached images, using:
+    ```
+    docker system prune -a
+    ```
+    [Optional] Modify submission.sh file if your agent needs any custom modifications (e.g. command-line arguments). Otherwise, nothing to do. Default submission.sh is simply a call to agent in `agent.py`
+
+1. Download all the required data into the `home-robot/data` directory (see [Habitat OVMM readme](projects/habitat_ovmm/README.md)). Then in your `docker run` command mount `home-robot/data` data folder to the `home-robot/data` folder in the Docker image (see `./scripts/test_local.sh` for reference).
+     
+1. Evaluate your docker container locally:
+    ```bash
+    ./scripts/test_local.sh --docker-name ovmm_baseline_submission
+    ```
+    If the above command runs successfully you will get an output similar to:
+    ```
+    Arguments:
+    {
+        "habitat_config_path": "ovmm/ovmm_eval.yaml",
+        "baseline_config_path": "projects/habitat_ovmm/configs/agent/hssd_eval.yaml",
+        "opts": []
+    }
+    ----------------------------------------------------------------------------------------------------
+    Configs:
+
+    ----------------------------------------------------------------------------------------------------
+    pybullet build time: May 20 2022 19:45:31
+    2023-07-03 15:04:05,629 Initializing dataset OVMMDataset-v0
+    2023-07-03 15:04:06,094 initializing sim OVMMSim-v0
+    2023-07-03 15:04:08,686 Initializing task OVMMNavToObjTask-v0
+    Running eval on [1200] episodes
+    Initializing episode...
+    [OVMM AGENT] step heuristic nav policy
+    Executing skill NAV_TO_OBJ at timestep 1
+    [OVMM AGENT] step heuristic nav policy
+    Executing skill NAV_TO_OBJ at timestep 2
+    [OVMM AGENT] step heuristic nav policy
+    Executing skill NAV_TO_OBJ at timestep 3
+    [OVMM AGENT] step heuristic nav policy
+    Executing skill NAV_TO_OBJ at timestep 4
+    [OVMM AGENT] step heuristic nav policy
+    Executing skill NAV_TO_OBJ at timestep 5
```

Comment:
Can we show what is the final output that people are supposed to see at the end?

Reply:
I've never run evaluation till the end. I asked @yvsriram to implement a random agent it would make it possible to run evaluation fast.

Reply:
Will add today.

Reply:
We can add this in the next PR